packages/components/psammead-navigation/src/index.jsx

```diff
@@ -217,9 +217,19 @@ NavigationLi.defaultProps = {
   dir: 'ltr',
 };
 
+// ampOpenClass is the class added to the Navigation, and is toggled on tap.
+// It indicates whether the menu is open or not. This overrides the background
+// color of the Navigation
 const StyledNav = styled.nav`
   position: relative;
   ${({ isOpen }) => `background-color: ${isOpen ? C_EBON : C_POSTBOX};`}
+  ${({ ampOpenClass }) =>
+    ampOpenClass &&
+    css`
+      &.${ampOpenClass} {
```

Comment:
Just wondering if we need this `&`?

Reply:
The `&` is required because without it we will be referring to the children of the navigation, and not the navigation, and then the background colour override would not work.

Reply:
My understanding is that you want to apply the `background-color: ${C_EBON};` when `ampOpenClass` is passed to the `StyledNav`, right? In that case, I think we don't need `&.${ampOpenClass} {`

Reply:
Yes, you're right. It works without `&.${ampOpenClass}`. Made that change